src/tests/integration-test

```diff
@@ -476,6 +478,17 @@ class UDisksTestCase(unittest.TestCase):
         print('Set up test device: r/w: %s, r/o: %s' % (rw_dev, ro_dev))
         return (rw_dev, ro_dev)
 
+    @classmethod
+    def teardown_cd_vdev(cls, cd_device):
+        """remove attached pktcdvd device from a scsi_debug cdrom device"""
+        if os.path.exists(cd_device) and find_executable("pktsetup"):
```

Comment:
I think it would be nice to get some output if `pktsetup` is not found, but the device exists.

Reply:
So the `cd_device` here is the cd-rom device, while we're trying to tear down the `/dev/pktcdvd/xx` device. There's no direct naming relationship between the two and we would have to look perhaps somewhere in `/sys`. Also, it is the `pktsetup` who sets up the `/dev/pktcdvd/` device via udev rules, so if there's no `pktsetup` in the system, there's most likely no attached pktcdvd device either. Somebody would have to `modprobe pktcdvd` and use `/dev/pktcdvd/control` directly to create a new device.

Hmm, do you want me to add comment with this explanation somewhere?

Reply:
> Hmm, do you want me to add comment with this explanation somewhere?

Sounds like a good idea to me. I think it's okay to say that the `pktsetup` is also setting the device up.

Reply:
Okay, added.